evennia/server/session.py

```diff
@@ -87,6 +88,10 @@ def init_session(self, protocol_key, address, sessionhandler):
                                "INPUTDEBUG": False,
                                "RAW": False,
                                "NOCOLOR": False}
+
+        if override_flags:
+            self.protocol_flags.update(override_flags)
```

Comment:
I think it's fine to simply do `self.protocol_flags["ENCODING"] =  _TELNET_ENCODING` here. Maybe add a comment to explain it.